pkg/target/target.go

```diff
@@ -264,6 +264,12 @@ func (h *K8sValidationTarget) HandleViolation(result *types.Result) error {
 	return nil
 }
 
+func propsWithDescription(props apiextensions.JSONSchemaProps, description string) apiextensions.JSONSchemaProps {
```

Comment:
I concur with the linter (pass by pointer)

Reply:
Done